src/core/Akka/IO/IO.cs

```diff
@@ -0,0 +1,16 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="IO.cs" company="Akka.NET Project">
+//     Copyright (C) 2009-2015 Typesafe Inc. <http://www.typesafe.com>
+//     Copyright (C) 2013-2015 Akka.NET project <https://github.com/akkadotnet/akka.net>
+// </copyright>
+//-----------------------------------------------------------------------
+
+using Akka.Actor;
+
+namespace Akka.IO
+{
+    public abstract class Extension : IExtension
```

Comment:
My suggestion is to rename this to `IOExtension` since while this is clearly logical in scope of the Akka.IO, the name could be missleading for the people using intelli sense while typing `Extension` when using akka.